vendor/phantomjs-launcher.js

```diff
@@ -0,0 +1,58 @@
+'use strict'
+var system = require('system')
+var fs = require('fs')
+var page = require('webpage').create()
+var url = system.args[1]
+page.viewportSize = {
+  width: 1024,
+  height: 768
+}
```

Comment:
Is there an issue raised to make these configurable?  I remember talking with Seena about that before